packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -202,12 +206,11 @@ class _TimePickerHeader extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    ThemeData theme = Theme.of(context);
-    TextTheme headerTheme = theme.primaryTextTheme;
-
+    ThemeData themeData = Theme.of(context);
```

Comment:
There are currently 10 occurrences of `themeData = Theme.of(context)` and 10 of `theme = Theme.of(context)` in the code base, but fair and balanced is probably not what we're shooting for. Since I think I've contributed to both sides of the issue, I'm not complaining either :-).